.github/workflows/publish-staging.yml

```diff
@@ -0,0 +1,46 @@
+name: Publish Staging Docker image
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  docker-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-cache
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USER }}
+          password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        id: docker_build
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          push: true
+          # Target production image from multi-stage build (check Dockerfile)
+          target: python-base
+          tags: gnosispm/safe-config-service:latest
```

Comment:
@Uxio0 images built on the `main` branch will be published to Docker Hub with the `latest` tag. Let me know if we should change it to something else

Reply:
That's reasonable

Reply:
Actually just synced with @giacomolicari and this was changed to `staging` to be in line with the other services 👍

Reply:
Makes sense. But they are just tags, you can even push both